python/gtsam/preamble/geometry.h

```diff
@@ -10,14 +10,8 @@
  * Without this they will be automatically converted to a Python object, and all
  * mutations on Python side will not be reflected on C++.
  */
-#include <pybind11/stl.h>
 
-PYBIND11_MAKE_OPAQUE(
-    std::vector<gtsam::Point2, Eigen::aligned_allocator<gtsam::Point2>>);
-PYBIND11_MAKE_OPAQUE(gtsam::Point2Pairs);
-PYBIND11_MAKE_OPAQUE(gtsam::Point3Pairs);
-PYBIND11_MAKE_OPAQUE(gtsam::Pose2Pairs);
-PYBIND11_MAKE_OPAQUE(gtsam::Pose3Pairs);
-PYBIND11_MAKE_OPAQUE(std::vector<gtsam::Pose3>);
-PYBIND11_MAKE_OPAQUE(gtsam::CameraSet<gtsam::PinholeCamera<gtsam::Cal3Bundler>>);
 PYBIND11_MAKE_OPAQUE(gtsam::CameraSet<gtsam::PinholeCamera<gtsam::Cal3_S2>>);
+PYBIND11_MAKE_OPAQUE(gtsam::CameraSet<gtsam::PinholeCamera<gtsam::Cal3Bundler>>);
```

Comment:
nit: could we add a small comment here explaining why List's can be converted automatically, but Set's cannot?

Reply:
Sets can be, and `CameraSet` is actually derived from `std::vector` (and poorly named IMO). Let me update the `test_Triangulation.py` file tests and see if I can get rid of these opaque markers.